tests/test_plots.py

```diff
@@ -47,6 +48,10 @@ def generate_base_data(seed=31):
     prior_predictive = rng.normal(size=(1, 100, 7))
     posterior_predictive = rng.normal(size=(4, 100, 7))
     diverging = rng.choice([True, False], size=(4, 100), p=[0.1, 0.9])
+    mu_prior_sampled = rng.normal(size=(1, 500))
+    tau_prior_sampled = np.exp(rng.normal(size=(1, 500)))
+    theta_prior_sampled = rng.normal(size=(1, 500, 8))
```

Comment:
```suggestion
    mu_prior = norm(0, 3).rvs(size=(1, 500), random_state=rng)
    tau_prior = halfnorm(0, 5).rvs(size=(1, 500),  random_state=rng)
    theta_prior = norm(0, 1).rvs(size=(1, 500),  random_state=rng)
```

Rename mu_prior, tau_prior and theta_prior to log_mu_prior, log_tau_prior and log_theta_prior